adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -126,67 +126,87 @@ - (BOOL)addDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
   if (delegate.adapterAdType == GADMAdapterVungleAdTypeInterstitial ||
       delegate.adapterAdType == GADMAdapterVungleAdTypeRewarded) {
     @synchronized(_delegates) {
-      if (delegate && ![_delegates objectForKey:delegate.desiredPlacement]) {
+      if (![_delegates objectForKey:delegate.desiredPlacement]) {
         GADMAdapterVungleMapTableSetObjectForKey(_delegates, delegate.desiredPlacement, delegate);
       }
     }
   } else if ([delegate respondsToSelector:@selector(isBannerAd)] && [delegate isBannerAd]) {
     @synchronized(_bannerDelegates) {
       // We only support displaying one Vungle Banner Ad at the same time currently
-      if (_bannerPlacementID != nil && ![_bannerPlacementID isEqualToString:delegate.desiredPlacement]) {
+      if (_bannerRequest != nil && ![_bannerRequest isEqualToBannerRequest:delegate.bannerRequest]) {
         return NO;
       }
 
-      if (delegate && ![_bannerDelegates objectForKey:delegate.bannerRequest]) {
+      if (![_bannerDelegates objectForKey:delegate.bannerRequest]) {
         NSEnumerator *enumerator = _bannerDelegates.keyEnumerator;
         GADMAdapterVungleBannerRequest *bannerRequest = nil;
```

Comment:
Just pointing this out - you've declared an object `_bannerRequest` as part of the `@implementation` above, and there is a local var declared here with the same name `bannerRequest`. I'd differentiate these somehow, even the tiniest bit, to prevent confusion in usage.

Reply:
@crbinsf Got it, thanks a lot for the guidance, I will change all the local vars `bannerRequest` to a different name. :)

Reply:
@crbinsf Done, thanks a lot. 😀